src/clarity_vm/clarity.rs

```diff
@@ -385,17 +385,18 @@ impl ClarityInstance {
             header_db,
             burn_state_db,
             cost_track,
-            mainnet: false,
+            mainnet: self.mainnet,
             epoch,
         };
 
+        let use_mainnet = self.mainnet.clone();
```

Comment:
`.clone()` should be unnecessary, since `bool` implements `Copy`.

Reply:
done